payments/utils.py

```diff
@@ -173,3 +173,45 @@ def check_payments_status(verbose=0):
 
         for payment in order_payments:
             _process_payment(order, payment, verbose)
+
+
+def send_unpaid_order_email_notifications(verbose=0):
+    """
+    Send email to all users who has unpaid orders.
+    """
+    deadline = timezone.now() - timedelta(days=UNPAID_ORDER_NOTIFICATION_DAYS)
+
+    orders = Order.objects.filter(Q(status=Order.AWAITING) | Q(status=Order.PARTLY_PAID))
+    orders = orders.filter(date_created__lt=deadline, unpaid_notification_sent_at__isnull=True)
```

Comment:
Lets update order to fileter for `unpaid_notification_sent` set to NULL or `unpaid_notification_sent` is lass than `UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY` and `unpaid_notification_sent_amount` is less than `UNPAID_ORDER_NOTIFICATION_REPEAT`